README.md

```diff
@@ -342,8 +342,8 @@ small.
 # Examples
 
 For using the Triton Python client in these examples you need to install
-the [Triton Python Client Library](https://github.com/triton-inference-server/server/blob/master/docs/client_libraries.md#getting-the-client-libraries).
-The Python client for each of the examples is in the `client.py` file.
+the [Triton Python Client Library](https://github.com/triton-inference-server/client#getting-the-client-libraries-and-examples).
```

Comment:
Please only include this change in your PR.

Reply:
@Tabrizian I updated my commit.

Reply:
Thanks, can you add "The Python client for each of the examples is in the `client.py` file."  back too?

Reply:
I didn't delete that sentence. You can check it in my commit.
I don't know why that sentence is minus.